### PR TITLE
chore: Report file - add more info on failed thresholds

### PR DESCRIPTION
**Pull request by mp-dthr**

### CODE REVIEW

**Comment by mp-dthr** [comment]
@oleiade / @na-- , any update? 🙏

File: junit.js [hunk 1]
```diff
@@ -32,28 +32,53 @@ function generateJUnitXML(data, options) {
...
+          failureMessage =
+            '"><failure message="failed, average value: ' +
+            metric.values.avg +
+            ', min value: ' +
+            metric.values.min +
+            ', mean value: ' +
+            metric.values.med +
+            ', max value: ' +
+            metric.values.max +
+            '"/></testcase>';
```
Comment (na--) [inline junit.js:71]: The rest of the PR LGTM, but this part might be problematic. Depending on the value of the [`summaryTrendStats` option](https://k6.io/docs/using-k6/k6-options/reference/#summary-trend-stats), some of these values might not be present in `metric.values`. To be fair, the same thing applies to the `metric.values.med` from the old code, so this was already a problem :disappointed: 

I think we should just programatically build the string here, somewhat like this:
```suggestion
            '"><failure message="failed, ' + 
            Object.entries(metric.values).map(function(kv) {return kv[0] + ' value: ' + kv[1];}).join(', ') +
            '"/></testcase>';
```

In fact, we can probably do this for all metric types, then we won't even need to have the `if (metric.type == 'sth') {}` chain :thinking:

Comment (mp-dthr) [reply junit.js:71]: I wasn't aware of this option. Fully agree 💯 . Feel free to update the PR. 🙇

Comment (mp-dthr) [reply junit.js]: Updated the PR, please review @na-- / @oleiade 🙇

**Comment by mp-dthr** [comment]
@na-- / @oleiade Review, please 🙇

**Comment by mp-dthr** [comment]
@na-- , please take a look 🙇

**Review by na-- — APPROVED** [review]

**Comment by mp-dthr** [comment]
Thank you guys @oleiade / @na-- ! 🙇 
When will the new 0.0.4 release be published?

**Comment by mp-dthr** [comment]
@oleiade / @na-- Seems like the 0.0.4 version was not published yet?

**Comment by oleiade** [comment]
Hi @mp-dthr so sorry we took so long to get back to you. There's been some internal changes going on, and it appears we lost track of this.

I will proceed with the `0.0.4` release immediately, and will let you know when it's published 🙇🏻

**Comment by oleiade** [comment]
I have published the version 0.0.4 on the `jslib.k6.io` repository. Time for the publishing action to complete, and you should be able to use the new version. Let me know if anything doesn't work as expected 🤝

**Pull request merged**